src/components/pages/dashboard/dashboard.tsx

```diff
@@ -182,7 +182,10 @@ const Dashboard: FC = () => {
       };
     }
 
-    if (artifactKind & UIArtifactKind.WORKS) {
+    const workArtefactsSelected = artifactKind & UIArtifactKind.WORKS;
```

Comment:
Befindet man sich aktuell in der Sammlungsansicht, sollen die Tabellenspalten genutzt werden, die für Nicht-Literatur-Artefakten definiert sind.